ansible/add-fqdn.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Add FQDN to hosts
+  hosts: all
+  gather_facts: false
+  vars_files:
+    - vars/defaults.yml
+  tasks:
+    - name: Ensure OS FQDN are added to /etc/hosts
+      ansible.builtin.lineinfile:
+        path: "/etc/hosts"
+        line: "{{ item }}"
+      loop:
+        - "192.168.37.2 internal.infra.mos.{{ root_domain }}"
+        - "192.168.38.2 external.infra.mos.{{ root_domain }}"
```

Comment:
What is external vs public?

Reply:
No reason for keeping `external` as all APIs are either on the `internal` or `public` interfaces.